.coderabbit.yaml

```
@@ -0,0 +1,23 @@
+  auto_review:
+    enabled: true
+    drafts: false
+
+    branches:
+      include:
+        - "main"
+
+chat:
+  auto_reply: true
+
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (2 > 0)

(empty-lines)